### PR TITLE
[PPML] Change Occlum image into install_package installation

### PR DESCRIPTION
**Pull request by qiyuangong**

### CODE REVIEW

**Comment by qiyuangong** [comment]
Address #3747

**Comment by qiyuangong** [comment]
PR blocked by https://github.com/guzongmin/occlum/commit/3a3082980743cbc67fd73c77c4a16a2d8b545dcc#diff-847918b1cf40151da777261492f12f0c333a573fa28676451865fa7d4ca758ee

**Comment by qiyuangong** [comment]
Local test passed, thanks to @ManfeiBai

File: ppml/trusted-realtime-ml/scala/docker-occlum/Dockerfile [hunk 1]
```diff
@@ -98,20 +98,41 @@ RUN mkdir -p /opt/analytics-zoo
...
+        gnupg2 vim jq make gdb libfuse-dev libtool \
+        libsgx-dcap-ql libsgx-epid libsgx-urts libsgx-quote-ex libsgx-uae-service \
+        libsgx-dcap-quote-verify-dev \
+        occlum && \
     apt-get clean
 
+RUN echo "source /etc/profile" >> $HOME/.bashrc
+
+# Hotfix for occlum mount aesm
+RUN sed -i "s|pgrep \"|if [ -S /var/run/aesmd/aesm.socket ]; then\n            return\n        else\n            pgrep \"|g" /opt/occlum/build/bin/occlum && \
```
Comment (glorysdj) [inline ppml/trusted-realtime-ml/scala/docker-occlum/Dockerfile:132]: these RUN can be merge with &&, which will reduce image size

Comment (qiyuangong) [reply ppml/trusted-realtime-ml/scala/docker-occlum/Dockerfile:132]: Yes. Has merged 2 sed into 1. Do we need to merge it with other bash comand?

Comment (qiyuangong) [reply ppml/trusted-realtime-ml/scala/docker-occlum/Dockerfile]: Already addressed. :)

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**